pkg/resources/validation/feature/feature.go

```diff
@@ -0,0 +1,63 @@
+package feature
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	objectsv3 "github.com/rancher/webhook/pkg/generated/objects/management.cattle.io/v3"
+	"github.com/rancher/wrangler/pkg/webhook"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/trace"
+)
+
+func NewValidator() webhook.Handler {
+	return &featureValidator{}
+}
+
+type featureValidator struct{}
+
+func (fv *featureValidator) Admit(response *webhook.Response, request *webhook.Request) error {
+	listTrace := trace.New("featureValidator Admit", trace.Field{Key: "user", Value: request.UserInfo.Username})
+	defer listTrace.LogIfLong(2 * time.Second)
+
+	if request.Operation == admissionv1.Create {
+		response.Result = &metav1.Status{
+			Status:  "Failure",
+			Message: "Cannot create new Features",
+			Reason:  metav1.StatusReasonMethodNotAllowed,
+			Code:    http.StatusMethodNotAllowed,
+		}
+		response.Allowed = false
+		return nil
+	}
+	if request.Operation == admissionv1.Delete {
+		response.Result = &metav1.Status{
+			Status:  "Failure",
+			Message: "Cannot delete Features",
+			Reason:  metav1.StatusReasonMethodNotAllowed,
+			Code:    http.StatusMethodNotAllowed,
+		}
+		response.Allowed = false
+		return nil
+	}
+
+	newFeature, oldFeature, err := objectsv3.FeatureOldAndNewFromRequest(request)
+	if err != nil {
+		return err
+	}
+	if oldFeature.Status.LockedValue != nil {
```

Comment:
This check is incomplete, what if the update is to the annotations? Or description? Saying 'No updates every' because the value is locked is a bad policy.